### PR TITLE
Add activity indicator for the ghost ui

### LINKED ISSUE #211: Add LargeProfileSummaryView ghost UI
Screenshot 2024-04-17 at 13 38 04

https://www.figma.com/file/AzU51EzjGYFJPtLKG2GBIr/SDK-Components?type=design&node-id=8870%3A13260&mode=design&t=lAQrgXEvgO4IJUl3-1

### PR DESCRIPTION
**Pull request by pinarol**
Closes https://github.com/Automattic/Gravatar-SDK-iOS/issues/211 https://github.com/Automattic/Gravatar-SDK-iOS/issues/210 https://github.com/Automattic/Gravatar-SDK-iOS/issues/209 https://github.com/Automattic/Gravatar-SDK-iOS/issues/208 

### Description

Adding the activity indicator for the ghost ui. It will animate the colors of the placeholders.

### Testing Steps

Demo app > Profile card > Test activity indicator switch

### CODE REVIEW

**Review by etoledom — APPROVED** [review]
Works and looks great 🎉

File: Tests/GravatarUITests/__Snapshots__/ProfileViewTests/testProfileViewLoadingStateClearsWhenDataIsPresent.light.png
Comment (etoledom) [inline Tests/GravatarUITests/__Snapshots__/ProfileViewTests/testProfileViewLoadingStateClearsWhenDataIsPresent.light.png:1]: By the way, I noticed also on the demo app that the account buttons are not visible 🤔

**Review by etoledom — CHANGES_REQUESTED** [review]
This is probably something to look at before merging. I just noticed now

**Comment by pinarol** [comment]
I believe i fixed it. @etoledom

**Review by etoledom — APPROVED** [review]
Thank you! 🙏

**Pull request merged**